MAVProxy/mavproxy.py

```diff
@@ -9,13 +9,31 @@
 
 import sys, os, time, socket, signal
 import fnmatch, errno, threading
-import serial, Queue, select
+import serial, select
 import traceback
 import select
 import shlex
 import platform
 import json
 
+try:
+    # Python 2.7
+    reload
```

Comment:
python-future.org seems to indicate `imp` under Python 2 contains reload... this would simplify this code a chunk by not doing the NameError check.  I'm uncomfortable with executing reload to test if it's a valid symbol :-)

http://python-future.org/compatible_idioms.html